src/components/FormSignIn.vue

```diff
@@ -48,12 +48,17 @@
 						>Lost Password?</a
 					>
 				</div>
-				<button
-					@click="onSubmit"
-					class="w-full rounded-lg bg-primary-700 px-5 py-3 text-center text-base font-medium text-white hover:bg-primary-800 focus:ring-4 focus:ring-primary-300 dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800 sm:w-auto"
-				>
-					Login to your account
-				</button>
+				<div>
+					<button
+						type="submit"
+						class="w-full rounded-lg bg-primary-700 px-5 py-3 text-center text-base font-medium text-white hover:bg-primary-800 focus:ring-4 focus:ring-primary-300 dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800 sm:w-auto"
+					>
+						Login to your account
+					</button>
+					<span v-if="errorMessage" class="mb-4 ml-2 w-full text-xs text-red-600">{{
+						errorMessage
+					}}</span>
```

Comment:
Hahaha, most pretty error message : ) Maybe we could use the alert from [Flowbite](https://flowbite.com/docs/components/alerts/#alerts-with-icon)

```suggestion
					<div
						v-if="errorMessage"
						class="mt-6 flex items-center rounded-lg bg-red-50 p-4 text-sm text-red-800 dark:bg-gray-700 dark:text-red-400"
						role="alert"
					>
						<svg
							class="mr-3 inline h-4 w-4 flex-shrink-0"
							aria-hidden="true"
							xmlns="http://www.w3.org/2000/svg"
							fill="currentColor"
							viewBox="0 0 20 20"
						>
							<path
								d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5ZM9.5 4a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3ZM12 15H8a1 1 0 0 1 0-2h1v-3H8a1 1 0 0 1 0-2h2a1 1 0 0 1 1 1v4h1a1 1 0 0 1 0 2Z"
							/>
						</svg>
						<span class="sr-only">Info</span>
						<div><span class="font-medium">Failure!</span> {{ errorMessage }}</div>
					</div>
```